operate/importer/src/main/java/io/camunda/operate/zeebeimport/ZeebeImporter.java

```diff
@@ -16,13 +16,11 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.context.annotation.DependsOn;
 import org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler;
 import org.springframework.stereotype.Component;
 
 @Component
 @Configuration
-@DependsOn("schemaStartup")
 public class ZeebeImporter {
```

Comment:
❓ now, this too will `scheduleReaders` and `start importing` without schema. is that safe? is it needed in operate?

Reply:
Same here. We need to test. I'm not sure why we put this condition.